api/Element.json

```diff
@@ -6263,6 +6263,53 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "accepts_TrustedHtml": {
```

Comment:
As for other cases, you can set the property with this, but if you were to get it back you'd get a string.

Reply:
That would be good. For now I have tried to be consistent.